### ISSUE TITLE
[MVT] Client side zoom for vector tiles

### ISSUE THREAD

**Issue by davidoesch** [geoadmin/mf-geoadmin3#4860]
all os all browsers

to improve the user experience :  add similiar as  for raster tiles a client side zooming for vector tiles.
Meaning: to overcome the white square whilst start rendering of the next zoomlevel
result: as in mapbox , the zooming in and out is more fluid, adaptive

-> pls check with ol6 community

**Comment by davidoesch** [comment]
This flickering is really annoying ... linked to https://github.com/geoadmin/mf-geoadmin3/issues/4861

**Comment by boecklic** [comment]
slightly improved zooming experience: https://mf-geoadmin4.int.bgdi.ch/ol-minimalist/index.html

**Comment by davidoesch** [comment]
Hmmm not really maybe b related to the connection here at DBX

**Comment by boecklic** [comment]
**tl;dr**: same behaviour as mapbox is currently not possible for performance reasons

mapbox features a "vectorial" zoom, meaning it smoothly scales/interpolates vector information between feature definitions on the exact zoom-level requested by the UI (e.g. 8.7). OL on the other hand renders features at fixed zoom levels (e.g. 8 and 9) and simply scales the rasterised image in between (e.g. for 8.7). This behaviour was improved with [OL-PR #9357](https://github.com/openlayers/openlayers/pull/9357) but will always be different to mapbox (i.e. the "roundabout" issue is currently not solvable).

**Comment by danduk82** [comment]
this will be solved (hopefully) by intergrating mapbox renderer as composite layer

**Comment by pakb** [comment]
Has been done
